extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -134,15 +134,19 @@ public void initialize() {
             stringChannelUID = new ChannelUID(getThing().getUID(), CHANNEL_STRING);
             try {
                 Channel stringChannel = getThing().getChannel(stringChannelUID.getId());
-                Configuration cfg = stringChannel.getConfiguration();
-                String dateTimeFormatString = (String) cfg.get(PROPERTY_DATE_TIME_FORMAT);
-                if (!(dateTimeFormatString == null || dateTimeFormatString.isEmpty())) {
-                    dateTimeFormat = new SimpleDateFormat(dateTimeFormatString);
+                if (stringChannel != null) {
+                    Configuration cfg = stringChannel.getConfiguration();
+                    String dateTimeFormatString = (String) cfg.get(PROPERTY_DATE_TIME_FORMAT);
+                    if (!(dateTimeFormatString == null || dateTimeFormatString.isEmpty())) {
+                        dateTimeFormat = new SimpleDateFormat(dateTimeFormatString);
+                    } else {
+                        logger.debug("No format set in channel config for {}. Using default format.", stringChannelUID);
+                        dateTimeFormat = new SimpleDateFormat(DATE_PATTERN_WITH_TZ);
+                    }
                 } else {
-                    logger.debug("No format set in channel config for {}. Using default format.", stringChannelUID);
-                    dateTimeFormat = new SimpleDateFormat(DATE_PATTERN_WITH_TZ);
+                    logger.debug("Missing channel: '{}'", stringChannelUID.getId());
                 }
-            } catch (Exception ex) {
+            } catch (RuntimeException ex) {

```

Comment:
Changed it on the go: We should not catch non runtime exception if there aren't any.